smtp-server/smtp-server.d.ts

```diff
@@ -0,0 +1,92 @@
+// Type definitions for smtp-server v1.15.0
+// Project: https://github.com/andris9/smtp-server
+// Definitions by: markisme <https://github.com/markisme/>
+// Definitions: https://github.com/markisme/smtp-server.git
```

Comment:
This should be `// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped`.